app/javascript/react/components/SessionsListView/SessionsListTile/SessionListTile.tsx

```diff
@@ -13,7 +13,7 @@ interface SessionListTile {
   id: number;
   sessionName: string;
   sensorName: string;
-  averageValue: number;
+  averageValue: number | "no data";
```

Comment:
hmm... I would use `averageValue?: number;` and later you can check if it's undefined